app/cdap/components/WranglerV2/SectionComponent/index.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import CheckCircleOutlinedIcon from '@material-ui/icons/CheckCircleOutlined';
```

Comment:
import react comes 1st
then remaining external library imports
finally the internal application file imports
please update across all files

Reply:
reordered imports